eve/service/svc_note.py

```diff
@@ -0,0 +1,172 @@
+import json
+from rich.console import Console
+from rich.table import Table
+from rich import box
+import os.path
+from datetime import date
+import pathlib
+
+
+class Note:
```

Comment:
I would try to structure your class like the below:

```python
class Note:
    def __init__(self):
        self.notes_filepath = os.path.join(pathlib.Path.home(), 'test.json')
        self.notes_file = json.loads(self.notes_filepath)
        
    def add_note(payload):
        """A method for adding a new note"""
        
    def delete_note(node_id):
        """A method to delete a note"""
        
    def edit_note(node_id, payload):
        """Overwrite a note with some payload"""

```

Reply:
Would you mind if I use `input` for creating new notes or altering existing notes? Maybe it is easier for the user to answer 3 input-questions than to put them to the command line like `eve note add '2021-06-01' 2 'Note Text'

I think of something like this:
`Enter a due date for the note (YYYY-MM-DD) [Default = None]: 2021-06-01
Enter a priority for the note (YYYY-MM-DD) [Default = None]: 2
Enter a note text for the note [Default = None]: Note Text`